build.gradle

```diff
@@ -47,6 +47,8 @@ dependencies {
     api "io.github.openfeign:feign-jackson:$openFeignVersion"
     api "io.github.openfeign:feign-slf4j:$openFeignVersion"
 
+    implementation "com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.13.0"
```

Comment:
Minor
```suggestion
    implementation "com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.13.2"
```

Reply:
I stayed with the version used by gradle to avoir version conflicts.
There is an entire subject on dependency management but I can't address it in this PR.